server/ott-config.ts

```diff
@@ -487,6 +487,17 @@ export function loadConfigFile() {
 	conf.validate({ allowed: "warn" });
 }
 
+export function validateConfig() {
+	try {
+		conf.validate({ allowed: "strict" });
+		validateMail();
```

Comment:
`validateMail` intentionally doesn't throw exceptions to not block startup. This is a feature. However, that also means that any validation errors will not be caught by this `try`.

I would recommend making `validateMail()` return a `Result` from `ott-common`, and return `err(msg)` whenever there is an error instead of logging it.